api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -121,6 +128,9 @@ message GroundStationConfiguration {
   // tracking a satellite (e.g., getting no signal at all), it can be useful to confirm these values
   // match the expected configuration of the antenna.
   antenna.AntennaConfiguration antenna = 3;
+
+  // The current configuration of the plan being executed by the ground station.
```

Comment:
Yeah, I was wondering about that too. Do you have a preference? It seems it could work either way.

I was thinking that if something is fixed during a plan, we call it a configuration (it's like a setting for the plan and something we know before the plan even starts).

From the perspective of the groundstation, you could also argue that 'plan state' also makes sense though since it updates every time the plan changes.

Reply:
Yeah while there isn't a reason to make it set in stone, the intent is configuration is returned once at the beginning of a plan for fixed parameters (for which TLE is one) while state is for real-time values that change and are sent continuously.